{"repository":"https://github.com/pm-str/questbot","dockerfilePath":"Dockerfile","startTime":1672963163215,"endTime":1672963232386,"clone":{"startTime":1672963163215,"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/pm-str/questbot'...\n","error":null,"endTime":1672963164021,"commit":"34c42a25e8400060a80e1a234ca7bc7a265769ae"},"originalBuild":{"startTime":1672963164112,"endTime":1672963232327,"error":{"code":1,"killed":false,"signal":null,"cmd":"docker build --no-cache --force-rm -t questbot:latest -f /tmp/dinghy-analysis/pm-str/questbot/Dockerfile ."},"stdout":"Sending build context to Docker daemon  375.8kB\r\r\nStep 1/12 : FROM python:3.6\n ---> 54260638d07c\nStep 2/12 : ENV env docker\n ---> Running in 2038d8917f40\nRemoving intermediate container 2038d8917f40\n ---> 522da9e0b179\nStep 3/12 : ENV DJANGO_SU_USERNAME root\n ---> Running in e768c3f167ef\nRemoving intermediate container e768c3f167ef\n ---> 5f5ebc5771f0\nStep 4/12 : ENV DJANGO_SU_EMAIL root@root.root\n ---> Running in c0f61aae86d0\nRemoving intermediate container c0f61aae86d0\n ---> 6e21bc187779\nStep 5/12 : ENV DJANGO_SU_PASS rootroot\n ---> Running in 726d2b1fe6e3\nRemoving intermediate container 726d2b1fe6e3\n ---> 2cf45054a30b\nStep 6/12 : RUN mkdir /source\n ---> Running in a2db0a19259d\nRemoving intermediate container a2db0a19259d\n ---> d909e45dbba5\nStep 7/12 : WORKDIR /source\n ---> Running in cc309cb9face\nRemoving intermediate container cc309cb9face\n ---> 4e2652cc12f4\nStep 8/12 : ADD Pipfile /source\n ---> 59c1f6a29b25\nStep 9/12 : ADD Pipfile.lock /source\n ---> 56686cd205be\nStep 10/12 : RUN pip install pipenv\n ---> Running in 7f79de004062\nCollecting pipenv\n  Downloading pipenv-2022.4.8-py2.py3-none-any.whl (3.6 MB)\nCollecting certifi\n  Downloading certifi-2022.12.7-py3-none-any.whl (155 kB)\nCollecting virtualenv\n  Downloading virtualenv-20.17.1-py3-none-any.whl (8.8 MB)\nRequirement already satisfied: pip>=18.0 in /usr/local/lib/python3.6/site-packages (from pipenv) (21.2.4)\nCollecting virtualenv-clone>=0.2.5\n  Downloading virtualenv_clone-0.5.7-py3-none-any.whl (6.6 kB)\nRequirement already satisfied: setuptools>=36.2.1 in /usr/local/lib/python3.6/site-packages (from pipenv) (57.5.0)\nCollecting filelock<4,>=3.4.1\n  Downloading filelock-3.4.1-py3-none-any.whl (9.9 kB)\nCollecting importlib-resources>=5.4\n  Downloading importlib_resources-5.4.0-py3-none-any.whl (28 kB)\nCollecting distlib<1,>=0.3.6\n  Downloading distlib-0.3.6-py2.py3-none-any.whl (468 kB)\nCollecting platformdirs<3,>=2.4\n  Downloading platformdirs-2.4.0-py3-none-any.whl (14 kB)\nCollecting importlib-metadata>=4.8.3\n  Downloading importlib_metadata-4.8.3-py3-none-any.whl (17 kB)\nCollecting zipp>=0.5\n  Downloading zipp-3.6.0-py3-none-any.whl (5.3 kB)\nCollecting typing-extensions>=3.6.4\n  Downloading typing_extensions-4.1.1-py3-none-any.whl (26 kB)\nInstalling collected packages: zipp, typing-extensions, platformdirs, importlib-resources, importlib-metadata, filelock, distlib, virtualenv-clone, virtualenv, certifi, pipenv\nSuccessfully installed certifi-2022.12.7 distlib-0.3.6 filelock-3.4.1 importlib-metadata-4.8.3 importlib-resources-5.4.0 pipenv-2022.4.8 platformdirs-2.4.0 typing-extensions-4.1.1 virtualenv-20.17.1 virtualenv-clone-0.5.7 zipp-3.6.0\n\u001b[91mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\n\u001b[0m\u001b[91mWARNING: You are using pip version 21.2.4; however, version 21.3.1 is available.\nYou should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.\n\u001b[0mRemoving intermediate container 7f79de004062\n ---> 3a8b70719537\nStep 11/12 : RUN pipenv install --system --deploy\n ---> Running in 09dc9cee25ef\nInstalling dependencies from Pipfile.lock (99e887)...\n\u001b[91mAn error occurred while installing markupsafe==1.0 --hash=sha256:a6be69091dac236ea9c6bc7d012beab42010fa914c459791d627dad4910eb665! Will try again.\n\u001b[0mInstalling initially failed dependencies...\n\u001b[91m[pipenv.exceptions.InstallError]: Collecting markupsafe==1.0\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]:   Using cached MarkupSafe-1.0.tar.gz (14 kB)\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]: ERROR: Command errored out with exit status 1:\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]:      command: /usr/local/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-r6nas8tu/markupsafe_b9acb5c7f07c4d04b2651917ed1c08ac/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-r6nas8tu/markupsafe_b9acb5c7f07c4d04b2651917ed1c08ac/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-t_5d4f7v\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]:          cwd: /tmp/pip-install-r6nas8tu/markupsafe_b9acb5c7f07c4d04b2651917ed1c08ac/\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]:     Complete output (5 lines):\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]:     Traceback (most recent call last):\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]:       File \"<string>\", line 1, in <module>\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]:       File \"/tmp/pip-install-r6nas8tu/markupsafe_b9acb5c7f07c4d04b2651917ed1c08ac/setup.py\", line 6, in <module>\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]:         from setuptools import setup, Extension, Feature\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]:     ImportError: cannot import name 'Feature'\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]:     ----------------------------------------\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]: WARNING: Discarding https://files.pythonhosted.org/packages/4d/de/32d741db316d8fdb7680822dd37001ef7a448255de9699ab4bfcbdf4172b/MarkupSafe-1.0.tar.gz#sha256=a6be69091dac236ea9c6bc7d012beab42010fa914c459791d627dad4910eb665 (from https://pypi.org/simple/markupsafe/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]: ERROR: Could not find a version that satisfies the requirement markupsafe==1.0 (from versions: 0.9, 0.9.1, 0.9.2, 0.9.3, 0.11, 0.12, 0.13, 0.14, 0.15, 0.16, 0.17, 0.18, 0.19, 0.20, 0.21, 0.22, 0.23, 1.0, 1.1.0, 1.1.1, 2.0.0a1, 2.0.0rc1, 2.0.0rc2, 2.0.0, 2.0.1)\n\u001b[0m\u001b[91m[pipenv.exceptions.InstallError]: ERROR: No matching distribution found for markupsafe==1.0\n\u001b[0m\u001b[91mERROR: Couldn't install package: markupsafe\n Package installation failed...\n\u001b[0mRemoving intermediate container 09dc9cee25ef\n","stderr":"The command '/bin/sh -c pipenv install --system --deploy' returned a non-zero code: 1\n"}}